tests/test_event.py

```diff
@@ -527,6 +529,39 @@ def test_deleting_an_event_does_not_exist(event_test_client, event):
     assert response.status_code == status.HTTP_404_NOT_FOUND
 
 
+def test_event_with_image(event_test_client, client, session):
+    img = Image.new("RGB", (60, 30), color="red")
+    img.save("pil_red.png")
```

Comment:
![Uploading image.png…]()

Reply:
I don't understand the comment , could you please elaborate?

Reply:
Oh, sorry, GitHub's fault :p

![image](https://user-images.githubusercontent.com/1492939/109014591-f453e480-76bc-11eb-85db-62eafc263815.png)